tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestSymbStateRewriterFoldSet.scala

```diff
@@ -7,7 +7,7 @@ import at.forsyte.apalache.tla.lir._
 import at.forsyte.apalache.tla.lir.oper.ApalacheOper
 
 trait TestSymbStateRewriterFoldSet extends RewriterBase {
-  private val types = Map(
+  Map(
```

Comment:
This is a fruitless value. Can scalafix detect expressions like that?

Reply:
It will eliminate unused expressions entirely when variables get bound to literals or known immutable values. But, it doesn't analyze all the way down into every construction to determine whether they are pure, so I think it's not safe just eliminate these values.